tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -333,3 +330,69 @@ def test_prometheus_pv_resize(self):
             pod_name=pod_name, pvc_name=pvc_name,
             filename="secondfilename", dirname="seconddirname",
             metric_data=resize_metrics, operation="create")
+
+    @pytest.mark.tier3
+    def test_prometheus_volume_metrics_on_node_reboot(self):
+        """Validate volume metrics using prometheus before and after node
+        reboot"""
+
+        # Create PVC and wait for it to be in 'Bound' state
+        pvc_name = self.create_and_wait_for_pvc()
+        pod_name = openshift_ops.oc_create_tiny_pod_with_volume(
+            self._master, pvc_name, "autotest-volume",
```

Comment:
instead of using constant string `autotest-volume` as prefix, generate string using random function and use it here.